scripts/install

```diff
@@ -1,117 +1,175 @@
+# Exit if an error occurs during the execution of the script
+ynh_abort_if_errors
 
-sudo yunohost app checkurl $domain$path -a $app
+#=================================================
+# RETRIEVE ARGUMENTS FROM THE MANIFEST
+#=================================================
 
-if [[ ! $? -eq 0 ]]; then
-    echo "Error : domain/path url isnt available"
-    exit 1
-fi
+app=$YNH_APP_INSTANCE_NAME
 
-####################################################
-#   Check that admin user is an existing account   #
-####################################################
+domain=$YNH_APP_ARG_DOMAIN
+path_url=$YNH_APP_ARG_PATH
+is_public=$YNH_APP_ARG_IS_PUBLIC
 
-sudo yunohost user list --json | grep -q "\"username\": \"$admin\""
-if [[ ! $? -eq 0 ]]; then
-    echo "Error : the chosen admin user does not exist"
-    exit 1
-fi
+#=================================================
+# CHECK IF THE APP CAN BE INSTALLED WITH THESE ARGS
+#=================================================
 
-sudo yunohost app setting $app admin -v $admin
+final_path=/var/www/$app
+test ! -e "$final_path" || ynh_die "This path already contains a folder"
 
-sudo yunohost app setting $app is_public -v "$is_public"
+# Normalize the url path syntax
+path_url=$(ynh_normalize_url_path "$path_url")
 
-####################################################
-#   Copy / install source files                    #
-####################################################
+# Check web path availability
+ynh_webpath_available "$domain" "$path_url"
+# Register (book) web path
+ynh_webpath_register "$app" "$domain" "$path_url"
 
-final_path=/var/www/$app
+#=================================================
+# STORE SETTINGS FROM MANIFEST
+#=================================================
 
-# Get and move sources
+ynh_app_setting_set "$app" domain "$domain"
+ynh_app_setting_set "$app" path   "$path_url"
+ynh_app_setting_set "$app" is_public "$is_public"
 
-tar xjvf ../sources/galette-$version.tar.bz2 > /dev/null
-sudo mv galette-$version/galette $final_path
-rm -r galette-$version
+#=================================================
+# INSTALL DEPENDENCIES
+#=================================================
 
-# Set permissions
+ynh_install_php --phpversion="$php_version" --package="$php_packages"
 
-sudo chown -R root:www-data $final_path
-sudo chmod -R o-rwx $final_path
-sudo chmod u+rx  $final_path/config
-sudo chmod g+rwx $final_path/config
-for folder in attachments cache exports files imports logs photos templates_c tempimages
-do
-    sudo chmod u+rx  $final_path/data/$folder
-    sudo chmod g+rwx $final_path/data/$folder
-done
+#=================================================
+# CREATE A MYSQL DATABASE
+#=================================================
+# If your app uses a MySQL database, you can use these lines to bootstrap
+# a database, an associated user and save the password in app settings
+
+db_name=$(ynh_sanitize_dbid "$app")
+db_pwd=$(ynh_string_random 15)
```

Comment:
we don't need to define the db password:
`ynh_mysql_setup_db` will create a database, an associated user and a ramdom password.